docs/Navigate-Pop/navigate-pop.md

```diff
@@ -0,0 +1,15 @@
+<!-- reference: 
+everything: https://support.system76.com/articles/pop-basics
+cosmic: https://github.com/pop-os/cosmic
+pop shell: https://github.com/pop-os/shell
+ubuntu vs. pop: https://support.system76.com/articles/difference-between-pop-ubuntu/
+-->
+
+# Navigate Pop!\_OS
+
+The Pop!\_OS Desktop environment is based on the GNOME shell and layered with other components called Pop!_Shell and C.O.S.M.I.C. (Computer Operating System Main Interface Components).
```

Comment:
`Pop!_Shell` probably needs an escape backslash for the underscore (`Pop!\_Shell`)

Reply:
Looking at the rendered page, it's not currently causing any problems. I can actually remove the escapes from the `Pop!_OS` strings and it still displays normally. mdbook uses [CommonMark](https://spec.commonmark.org/0.30/#emphasis-and-strong-emphasis), which (like GitHub and StackOverflow) uses slightly stricter parsing than default Markdown. https://spec.commonmark.org/0.30/#emphasis-and-strong-emphasis 

It might still be a good idea to escape the underscores as a matter of good Markdown practice, to ease switching to other engines in the distant future, and to better support text editors that don't understand CommonMark (`nano` highlights text as if it's going to be indented even when properly escaped with a backslash, though.)